src/webgpu/api/validation/createBindGroup.spec.ts

```diff
@@ -747,6 +747,59 @@ g.test('storage_texture,mip_level_count')
     }, mipLevelCount !== 1);
   });
 
+g.test('storage_texture,format')
+  .desc(
+    `
+    Test that the format of the storage texture is equal to resource's descriptor format if the
+    BindGroup entry defines storageTexture.
+  `
+  )
+  .params(u =>
+    u //
+      .combine('storageTextureFormat', [
+        'rgba8unorm',
+        'rg32float',
+        'r32sint',
+        'rgba16float',
+        'rgba8snorm',
+      ] as const)
```

Comment:
We should pull this list from `kAllTextureFormatInfo` with something like:

```ts
const kStorageTextureFormats = kAllTextureFormats.filter(f => kAllTextureFormatInfo[f].storage);
```

Reply:
Ah, right. This test should use the list. Fixed!